tests/inspection/test_info.py

```diff
@@ -252,20 +255,9 @@ def test_info_from_requires_txt(fixture_dir: FixtureDirGetter) -> None:
     demo_check_info(info)
 
 
-def test_info_from_setup_py(demo_setup: Path) -> None:
-    info = PackageInfo.from_setup_files(demo_setup)
-    demo_check_info(info, requires_dist={"package"})
-
-
-def test_info_from_setup_cfg(demo_setup_cfg: Path) -> None:
-    info = PackageInfo.from_setup_files(demo_setup_cfg)
-    demo_check_info(info, requires_dist={"package"})
-
```

Comment:
Should we drop these? or rather replace the `from_setup_files` call with `from_directory` or are these covered elsewhere in the suite?

Reply:
my thinking was that these tests were specifically aimed at the reading of setup.py and setup.cfg: and since that function has been removed they are no longer testing the thing that they say they are testing.

I am pretty sure that we are still erring on the side of having redundant tests rather than missing anything.  I have not (so far as I remember!) removed any testcases that were aimed at `from_directory()`, and quite a lot of testcases that previously would have been satisfied by the setup reader are now instead taking that path.

Reply:
I suspect that is a fair assumption for now, and this is something we can rectify later if we need to. Resolving.